ouroboros-network/src/Ouroboros/Network/BlockFetch/State.hs

```diff
@@ -238,7 +237,7 @@ data FetchNonTriggerVariables peer header block m = FetchNonTriggerVariables {
 data FetchStateFingerprint peer header block =
      FetchStateFingerprint
        (Maybe (Point block))
-       (Map peer (Maybe (Point header)))
+       (Map peer (Point header))
```

Comment:
So this will be the anchor point if the fragment is empty right?

Reply:
Exactly